docs/concepts/gardenlet.md

```diff
@@ -1,119 +1,137 @@
+* Provide a `seedSelector` that incorporates a label selector for the targeted seed clusters if you want the gardenlet to manage multiple seeds, see [the example gardenlet configuration](../../example/20-componentconfig-gardenlet.yaml#L68).
+In this case, you have to create the `Seed` objects together with a `kubeconfig` pointing to the cluster yourself.
 
 ## Component Configuration
 
-You can find an example configuration file [here](../../example/20-componentconfig-gardenlet.yaml).
+In the component configuration for the gardenlet, it’s possible to define:
 
-Basically, it is possible to define settings for the Kubernetes clients interacting with the various clusters, settings for the control loops inside the Gardenlet, settings for leader election and log levels, feature gates, and seed selection/configuration.
+* settings for the Kubernetes clients interacting with the various clusters
+* settings for the control loops inside the gardenlet
+* settings for leader election and log levels, feature gates, and seed selection or seed configuration.
 
-Most of the configuration options are similar to what the gardener-controller-manager offered with previous Gardener versions.
+More information: [Example Gardenlet Component Configuration](../../example/20-componentconfig-gardenlet.yaml).
 
 ## Heartbeats
 
-Similar to how Kubernetes is meanwhile using `Lease` objects for node heart beats (see [KEP](https://github.com/kubernetes/enhancements/blob/master/keps/sig-node/0009-node-heartbeat.md)), the Gardenlet is using `Lease`s objects for seed heart beats.
-Every two seconds it is checking its connectivity to the seed and then it renews its lease.
-The status will be reported in the `GardenletReady` condition in its `Seed` object(s).
-Similarly to the `node-lifecycle-controller` inside the kube-controller-manager, the gardener-controller-manager features a `seed-lifecycle-controller` that will set the `GardenletReady` condition to `Unknown` in case the Gardenlet stops sending its heartbeat signals.
-Carrying on, this will make the gardener-scheduler not considering this seed for newly created shoots anymore.
+Similar to how Kubernetes is meanwhile using `Lease` objects for node heart beats (see [KEP](https://github.com/kubernetes/enhancements/blob/master/keps/sig-node/0009-node-heartbeat.md)), the gardenlet is using `Lease` objects for heart beats of the seed cluster.
+Every two seconds, gardenlet is checking its connectivity to the seed cluster and then it renews its lease.
+The status is reported in the `GardenletReady` condition in its `Seed` object list.
+Similarly to the `node-lifecycle-controller` inside the `kube-controller-manager`, the `gardener-controller-manager` features a `seed-lifecycle-controller` that is set the `GardenletReady` condition to `Unknown` in case the gardenlet stops sending its heartbeat signals.
```

Comment:
```suggestion
Similarly to the `node-lifecycle-controller` inside the `kube-controller-manager`, the `gardener-controller-manager` features a `seed-lifecycle-controller` that sets the `GardenletReady` condition to `Unknown` in case the gardenlet stops sending its heartbeat signals.
```